diff --git a/t/lib/cygwin.t b/t/lib/cygwin.t
index ba86170..69502a3 100644
--- a/t/lib/cygwin.t
+++ b/t/lib/cygwin.t
@@ -61,7 +61,10 @@ unless ($prefix2) {
     chomp($prefix2 = `cygpath C:`);
     $prefix2 = substr($prefix2, 0, -1-(length($prefix2)>2));
 }
+SKIP: {
+skip("Occasional spurious failures on cygwin");
 is($prefix, $prefix2, 'cygdrive mount prefix2 = ' . $prefix2);
+}
 
 my @mnttbl = Cygwin::mount_table();
 ok(@mnttbl > 0, "non empty mount_table");
