--- makefile.mk	2016-07-16 16:09:01.553528800 -0400
+++ makefile.64.6.1.0.mk	2016-07-16 16:11:15.073080500 -0400
@@ -105,7 +105,7 @@
 # Uncomment this if you want to support the use of long doubles in GCC builds.
 # This option is not supported for MSVC builds.
 #
-#USE_LONG_DOUBLE *=define
+USE_LONG_DOUBLE *=define
 
 #
 # Uncomment this if you want to disable looking up values from
@@ -234,7 +234,7 @@
 # not be quoted)
 #
 .IF "$(CCTYPE)" == "GCC"
-CCHOME		*= C:\MinGW
+CCHOME		*= C:\mingw-w64\x86_64-6.1.0-win32-sjlj\mingw64
 .ELSE
 CCHOME		*= $(MSVCDIR)
 .ENDIF
