--- makefile.mk	2016-07-16 16:09:01.553528800 -0400
+++ makefile.32.static.mk	2016-07-16 16:11:30.064949100 -0400
@@ -33,7 +33,7 @@
 # Uncomment if you want to build a 32-bit Perl using a 32-bit compiler
 # on a 64-bit version of Windows.
 #
-#WIN64		*= undef
+WIN64		*= undef
 
 #
 # Comment this out if you DON'T want your perl installation to be versioned.
@@ -215,7 +215,7 @@
 # variables below. A static library perl525s.lib will also be created.
 # Ordinary perl.exe is not affected by this option.
 #
-#BUILD_STATIC	*= define
+BUILD_STATIC	*= define
 
 #
 # in addition to BUILD_STATIC the option ALL_STATIC makes *every*
@@ -224,7 +224,7 @@
 # is to have proper linking set so as to be able to create miscellaneous
 # executables with different built-in extensions
 #
-#ALL_STATIC	*= define
+ALL_STATIC	*= define
 
 #
 # set the install locations of the compiler include/libraries
