diff --git a/dist/Time-HiRes/t/ualarm.t b/dist/Time-HiRes/t/ualarm.t
index b50a175..6c40af0 100644
--- a/dist/Time-HiRes/t/ualarm.t
+++ b/dist/Time-HiRes/t/ualarm.t
@@ -98,7 +98,10 @@ for my $n (100_000, 1_100_000, 2_200_000, 4_300_000) {
     print("# t1 = $t1\n");
     printf("# t1 - t0 = %s\n", ($t1 - $t0));
     print("# got1 = $got1\n");
+    SKIP: {
+    skip "Fails in heavily loaded smokers;
     ok $got0 == 0 or print("# $got0\n");
+    }
     SKIP: {
        skip "alarm interval exceeded", 2 if $t1 - $t0 >= 0.5;
        ok $got1 > 0;
